docs/5.Policy Index/all.md

```diff
@@ -2300,9 +2300,9 @@ nav_order: 1
-| 2292 | CKV_GCP_112              | resource                         | google_kms_crypto_key_iam_binding                                                                | Esnure KMS policy should not allow public access                                                                                                                                                         | Terraform               | [GoogleKMSKeyIsPublic.py](https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/gcp/GoogleKMSKeyIsPublic.py)                                                                               |
-| 2293 | CKV_GCP_112              | resource                         | google_kms_crypto_key_iam_member                                                                 | Esnure KMS policy should not allow public access                                                                                                                                                         | Terraform               | [GoogleKMSKeyIsPublic.py](https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/gcp/GoogleKMSKeyIsPublic.py)                                                                               |
-| 2294 | CKV_GCP_112              | resource                         | google_kms_crypto_key_iam_policy                                                                 | Esnure KMS policy should not allow public access                                                                                                                                                         | Terraform               | [GoogleKMSKeyIsPublic.py](https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/gcp/GoogleKMSKeyIsPublic.py)                                                                               |
+| 2292 | CKV_GCP_112              | resource                         | google_kms_crypto_key_iam_binding                                                                | Ensure KMS policy should not allow public access                                                                                                                                                         | Terraform               | [GoogleKMSKeyIsPublic.py](https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/gcp/GoogleKMSKeyIsPublic.py)                                                                               |
```

Comment:
nonblocking: seems they are all CKV_GCP_112 related, is there a global variable or const to use to avoid duplicate messages?